tests/aws_iot_secure_tunneling_client_test.c

```diff
@@ -98,6 +104,15 @@ int main(int argc, char **argv) {
     ASSERT_SUCCESS(aws_condition_variable_wait(&condition_variable, &mutex));
     aws_mutex_unlock(&mutex);
 
+    char *payload = "Hi! I'm Paul / Some random payload";
+    struct aws_byte_buf buffer = aws_byte_buf_from_c_str(payload);
+    secure_tunnel->vtable.send_data(secure_tunnel, &buffer);
+
+    char *payload2 = "Hi! I'm Paul / Some random payload";
+    struct aws_byte_buf buffer2 = aws_byte_buf_from_c_str(payload2);
+    secure_tunnel->vtable.send_data(secure_tunnel, &buffer2);
+    // usleep(50000000);
```

Comment:
use `aws_thread_current_sleep` if you need to wait.

Reply:
Thanks, they're commented out here but I've replaced them in the stream start/reset pr

Reply:
You'll eventually want to use a condition variable and wait on a response, right?

Reply:
Yeah I could do what Francis did with the on_connection_complete, I just hadn't.